tutorials/fundamentals/create-element.md

```diff
@@ -0,0 +1,58 @@
+The `createElement()` function in JavaScript is used to programatically add elements to a web file.
+It takes one argument which is the type of element to create.
+You should store it in a variable as you will most likely want to manipulate the newly created element's properties.
+
+<div id="append-elements"></div>
+<button onclick="addElement()">Click to Add</button>
+
+<script>
+function addElement() {
+const doc = document.createElement('div');
+doc.innerHTML = 'Hello World';
+const descendant = document.querySelector('#append-elements');
+console.log(descendant)
+descendant.insertAdjacentElement('afterend', doc);
+}
+</script>
+
+```html
+<div id="append-elements"></div>
+<button onclick="addElement()">Click to Add</button>
+```
+
+```javascript
+function addElement() {
+const doc = document.createElement('div');
+doc.innerHTML = 'Hello World';
+const descendant = document.querySelector('#append-elements');
+console.log(descendant)
+descendant.insertAdjacentElement('afterend', doc);
+}
+```
+
+The more nested, the more appending needs to be done.
```

Comment:
"You can also use `appendChild()` on the newly created element to add more elements as shown below."